### PR TITLE
[NIFI-2050] [NIFI-2076] [NIFI-2136] [NIFI-2139] [NIFI-2134] [NIFI-2354] [NIFI-2368] [NIFI-2349] [NIFI-2351] [NIFI-2350] [NIFI-2367] [NIFI-2143]

### PR DESCRIPTION
**Pull request by scottyaslan**
-update dialog advanced buttons
-fix Add inline controller service dialog description width
-update provenance details dialog

### CODE REVIEW

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css [hunk 1]
```diff
@@ -270,11 +272,13 @@ div.modified-attribute-label {
 
 div.modified-attribute-value {
```
Comment (moranr) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css]: @scottyaslan please add `font-weight: 500;` and change `padding-top: 5px;` to `padding-top: 2px;`

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css [hunk 2]
```diff
@@ -178,31 +178,26 @@ div.flowfile-container {
     width: 282px;
     height: 120px;
```
Comment (moranr) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css]: @scottyaslan I think we can remove the `height` property all together so the Children FlowFiles heading will not sit so far below Parent FlowFile heading

Comment (moranr) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css]: @scottyaslan Also please add `line-height:1.4`. When multiples are listed they sit a little close to each other.

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/common-ui.css [hunk 3]
```diff
@@ -150,9 +150,8 @@ ul.links span.header-link-over {
 }
 
 .blank, .unset, .sensitive {
```
Comment (moranr) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/common-ui.css]: @scottyaslan Where `.unset` is being applied here, it needs `font-size: 13px` and `padding-top: 2px`
Not sure all places the other classes are referenced...

**Comment by mcgilman** [comment]
Thanks @scottyaslan! Looks good.. this has been merged to master.

**Pull request closed without merging**